test/server/config_validation/xds_verifier_test.cc

```diff
@@ -179,49 +179,68 @@ TEST(XdsVerifier, WarmMultipleListenersDelta) {
 TEST(XdsVerifier, ResendRouteSOTW) {
   XdsVerifier verifier(test::server::config_validation::Config::SOTW);
 
-  // send a route that will be ignored
+  // Send a route that will be ignored.
   verifier.routeAdded(buildRoute("route_config_0"));
 
-  // add a warming listener that refers to this route
+  // Add a warming listener that refers to this route.
   verifier.listenerAdded(buildListener("listener_0", "route_config_0"));
   EXPECT_TRUE(verifier.hasListener("listener_0", verifier.WARMING));
 
-  // send the same route again, make sure listener becomes active
+  // Send the same route again, make sure listener becomes active.
   verifier.routeAdded(buildRoute("route_config_0"));
   EXPECT_TRUE(verifier.hasListener("listener_0", verifier.ACTIVE));
 }
 
 TEST(XdsVerifier, ResendRouteDelta) {
   XdsVerifier verifier(test::server::config_validation::Config::DELTA);
 
-  // send a route that will be ignored
+  // Send a route that will be ignored.
   verifier.routeAdded(buildRoute("route_config_0"));
 
-  // add a warming listener that refers to this route
+  // Add a warming listener that refers to this route.
   verifier.listenerAdded(buildListener("listener_0", "route_config_0"));
   EXPECT_TRUE(verifier.hasListener("listener_0", verifier.WARMING));
 
-  // send the same route again, make sure listener becomes active
+  // Send the same route again, make sure listener becomes active.
```

Comment:
Nit: please split out these bulk formatting changes to distinct PRs in future, as they make it hard to review the substantive changes.